core-web/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 120
```

Comment:
Does this fix the indenting issue?

Reply:
Yes. `format:write` use prettier maybe the default is 80, I put 120 and did not touch the file.